src/handlers/google_redirect_handler.erl

```diff
@@ -0,0 +1,25 @@
+-module(google_redirect_handler).
```

Comment:
this looks pretty similar to github_redirect_handler, I bet you can parametrize the provider in the url, and move the parts that change (GOOGLE_CLIENTID, GoogleUrl) to configuration that you grab based on parameter

Reply:
You could _probably_ do the same with the callback handler, although I admit it's going to be more complicated and maybe not worth the effort if we don't plan to add more providers.